### DISCUSSION TITLE
supporting bun.js

### DISCUSSION THREAD

**Discussion by taheri24** [remix-run/remix#3687] [Q&A] [answered]
I had a great time with [Bun](https://bun.sh/) . It is a really fast and future-proof project.  I think if remix-js support it . 
It can really amazing 

also, I found a project for next with name  [bun-framework-next](https://www.npmjs.com/package/bun-framework-next).

**Comment by kiliman** [comment, marked as answer]
They've already started experimenting with it.

https://github.com/jacob-ebey/remix-bun-testing

Live demo: https://bun-remix-test.fly.dev/
